task4/style.css

```diff
@@ -0,0 +1,510 @@
+body {
+    font-family: "Roboto", sans-serif;
+}
+
+.background {
+    background: #FFFFFF;
+}
+
+h2 {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 24px;
+    color: #FFFFFF;
+}
+
+.column {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+.left-column {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    flex-direction: column;
+    width: 50%;
+}
+
+.right-column {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+    width: 50%;
+}
+
+.box-main-1 {
+    width: 344px;
+    height: 303px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 2px;
+    margin-right: 29px;
+    margin-top: 82px;
+}
+
+.box-1 {
+    width: 344px;
+    height: 175px;
+    background: #009688;
+    padding-top: 1px;
+}
+
+.box-1-text {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 24px;
+    margin-left: 18px;
+    margin-top: -3px;
+}
+
+
+.box-main-2 {
+    width: 344px;
+    height: 224px;
+    margin-top: 82px;
+    margin-left: 27px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 2px;
+}
+
+.box-2 {
```

Comment:
duplication of styles `.box-1`, create class `.box` and classes for background colors